lib/proxy.js

```diff
@@ -48,32 +46,7 @@ export default class PodiumProxy {
             );
         });
 
-        this.#registry = new Cache({
-            ttl: maxAge,
-        });
-        this.#registry.on('error', (error) => {
-            this.#log.error(
-                'Error emitted by the registry in @podium/proxy module',
-                error,
-            );
-        });
-        this.#registry.on('set', (key, item) => {
-            Object.keys(item.proxy).forEach((name) => {
-                const path = utils.pathnameBuilder(
-                    this.#pathname,
-                    this.#prefix,
-                    key,
-                    name,
-                );
-                this.#log.debug(
-                    `a proxy endpoint is mounted at pathname: ${path} pointing to: ${item.proxy[name]}`,
-                );
-            });
-        });
-
-        this.#registry.on('dispose', (key) => {
-            this.#log.debug(`dispose proxy item on key "${key}"`);
-        });
+        this.#registry = new Map();
```

Comment:
For my own learnings, why are we able to do this swap out?

Reply:
The `ttl-mem-cache` is cache module where objects can be stored for a period of time. Initially the idea was that a layout cold be very long living and podlets would be mounted in and possibly out of a layout and then having objects live in the cache with a timeout would make sure that when a podlet left a layout the timeout would clean out podlets not registered any more.

Though; this kinda never became the case and the cache have more or less never really used the time out feature (one can set object to live in the cache forever) so the cache is used just as a map lookup. Then, it can just be a map.